test/test_manifest.json

```diff
@@ -1521,6 +1521,19 @@
        "lastPage": 1,
        "type": "eq"
     },
+    {  "id": "issue6387",
+       "file": "pdfs/issue6387.pdf",
+       "md5": "08c39ac6d0aab1596e6e59793eaf3ee4",
+       "rounds": 1,
+       "type": "eq"
+    },
+    {  "id": "issue6387",
```

Comment:
The `id`s must be unique, otherwise the reference image for second test will replace the first one and thus cause test failures. Please change it to `"id": "issue6387-test"`.